pkg/resources/create_test.go

```diff
@@ -39,10 +39,6 @@ import (
 )
 
 const (
-	resourceLabel = triggers.GroupName + triggers.EventListenerLabelKey
-	triggerLabel  = triggers.GroupName + triggers.TriggerLabelKey
-	eventIDLabel  = triggers.GroupName + triggers.EventIDLabelKey
```

Comment:
are we not using these anywhere ?

Reply:
I see below you manually added value instead of the keeping these labels as it is and using everywhere would be nice